openreview/tools.py

```diff
@@ -1628,7 +1628,7 @@ def get_conflicts(author_profiles, user_profile, policy='default'):
         author_relations.update(author_info['relations'])
```

Comment:
Does it not need to pass n_years when getting the author info?

Reply:
I meant to link the comment to this line: author_info = info_function(profile)

Reply:
oops you are right!
I missed this part.